gtsam/geometry/Rot3.h

```diff
@@ -268,9 +270,13 @@ namespace gtsam {
     /// Syntatic sugar for composing two rotations
     Rot3 operator*(const Rot3& R2) const;
 
-    /// inverse of a rotation, TODO should be different for M/Q
+    /// inverse of a rotation
     Rot3 inverse() const {
-      return Rot3(Matrix3(transpose()));
+#ifdef GTSAM_USE_QUATERNIONS
+      return quaternion_.inverse();
+#else
+      return Rot3(transpose());
```

Comment:
Since you're changing it, why not Rot3(rot_.transpose()) ?

Reply:
So be more explicit? I figured I'd make minimal changes to the code.

Reply:
More efficient. And Rot3 constructor with one argument should be explicit.